lib/inspec/resources/filesystem.rb

```diff
@@ -1,3 +1,5 @@
+require 'inspec/resources/command'
+
 module Inspec::Resources
   class FileSystemResource < Inspec.resource(1)
```

Comment:
Rename this one to `FileSystem` ?

Reply:
Totally... but I agree with @clintoncwolfe that we should do renames in a separate PR. I have a stash somewhere that'll report all the renames needed on first load so it should be quick and easy.